src/pages/index.js

```diff
@@ -146,24 +202,107 @@ function closeModal(modal) {
   document.removeEventListener("keydown", handleEscapePress);
 }
 
+function handleDeleteSubmit(evt) {
+  evt.preventDefault();
+
+  const submitBtn = evt.submitter;
+  setButtonText(submitBtn, true, "Delete", "Deleting...");
+
+  api
+    .deleteCard(selectedCardId)
+    .then(() => {
+      selectedCard.remove();
+      closeModal(deleteModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      submitBtn.textContent = "Delete";
```

Comment:
` submitBtn.textContent = "Delete";`

Please, reuse the `setButtonText` function for setting the button text


```js
    .finally(() => {
      setButtonText(submitBtn, false, "Delete");
    });
```